tests/pipeline/test_pipeline_algo.py

```diff
@@ -181,6 +183,9 @@ def expected_close(self, date, asset):
             lookup = self.adj_closes
         return lookup.loc[date, asset]
 
+    def expected_volume(self, date, asset):
```

Comment:
I would expect this function to have to account for the split in the test data used by this test.